MyHeartCounts.xcodeproj/project.pbxproj

```diff
@@ -1772,6 +1772,14 @@
 				minimumVersion = 2.4.1;
 			};
 		};
+		801CA6182E6CCB2F006255C4 /* XCRemoteSwiftPackageReference "SpeziSensorKit" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/StanfordSpezi/SpeziSensorKit.git";
+			requirement = {
+				kind = revision;
+				revision = a0c9b64b5822c1d6aa7493a4da9e8229115462f2;
```

Comment:
TODO: switch to tagged release once https://github.com/StanfordSpezi/SpeziSensorKit/pull/5 is merged